atmo/jobs/forms.py

```diff
@@ -184,12 +186,6 @@ def __init__(self, *args, **kwargs):
 
     class Meta(BaseSparkJobForm.Meta):
         fields = BaseSparkJobForm.Meta.fields + ['emr_release']
```

Comment:
Yeah, that's still required since while the form field is still rendered when not adding this to the field list, the form validation would not happen for it since the form class inherits the form meta class from `BaseSparkJobForm`, which doesn't specify the `emr_release` field.